backend/tests/core/Entities/ProjectHelper.cs

```diff
@@ -62,6 +62,7 @@ public static Entity.Project CreateProject(int id, Entity.Agency agency = null,
                 PublicNote = $"publicNote-{id}",
                 PrivateNote = $"privateNote-{id}",
                 UpdatedOn = DateTime.UtcNow,
+                Metadata = "{offerAmount: 123}",
```

Comment:
Technically that isn't valid JSON, but I don't know if it'll cause any issue. `{"offerAmount": 123}`

Reply:
The test passes as-is.